src/test/java/fixture/MenuFixture.java

```diff
@@ -0,0 +1,56 @@
+package fixture;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.UUID;
+
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.domain.Product;
+
+public class MenuFixture {
+	private static final String VALID_MENU_GROUP_NAME = "점심특선";
+	private static final String VALID_MENU_NAME = "버거세트";
+	private static final int VALID_MENU_PRODUCT_QUANTITY = 2;
+	private static final BigDecimal VALID_MENU_PRICE = new BigDecimal("17.98");
+
+	public static MenuGroup createValidMenuGroup() {
+		UUID menuGroupId = UUID.randomUUID();
+		MenuGroup validMenuGroup = new MenuGroup();
+		validMenuGroup.setId(menuGroupId);
+		validMenuGroup.setName(VALID_MENU_GROUP_NAME);
+		return validMenuGroup;
+	}
+
+	public static MenuProduct createValidMenuProduct(int quantity) {
+		Product validProduct = ProductFixture.createValidProduct();
+
+		MenuProduct menuProduct = new MenuProduct();
+		menuProduct.setProduct(validProduct);
+		menuProduct.setProductId(validProduct.getId());
+		menuProduct.setQuantity(quantity);
+		return menuProduct;
+	}
+
+	public static Menu createValidMenu() {
+		MenuGroup menuGroup = createValidMenuGroup();
+
+		Menu menu = new Menu();
+		menu.setMenuGroupId(menuGroup.getId());
+		menu.setMenuGroup(menuGroup);
+		menu.setId(UUID.randomUUID());
+		menu.setName(VALID_MENU_NAME);
+		menu.setPrice(VALID_MENU_PRICE);
+		menu.setMenuProducts(
+			List.of(createValidMenuProduct(VALID_MENU_PRODUCT_QUANTITY)));
+		menu.setDisplayed(true);
+		return menu;
+	}
```

Comment:
상황에 따라서 지금 Fixture Class에서 정의되지 않은 객체를 생성해야할 수도 있을 것 같은데요. 👀
메서드 오버라이드를 잘 활용해서 이런 다양한 케이스에 대한 객체 생성도 쉽게할 수 있도록해보면 어떨까요? 😃

```suggestion
	public static Menu createValidMenu(String menuName, BigDecimal menuPrice, List<MenuProduct> menuProducts, boolean displayed) {
        MenuGroup menuGroup = createValidMenuGroup();

        Menu menu = new Menu();
        menu.setMenuGroupId(menuGroup.getId());
        menu.setMenuGroup(menuGroup);
        menu.setId(UUID.randomUUID());
        menu.setName(menuName);
        menu.setPrice(menuPrice);
        menu.setMenuProducts(menuProducts);
        menu.setDisplayed(displayed);
        return menu;
    }

    public static Menu createValidMenu() {
        return createValidMenu(
            VALID_MENU_NAME,
            VALID_MENU_PRICE,
            List.of(createValidMenuProduct(VALID_MENU_PRODUCT_QUANTITY)),
            true
        );
    }
```